src/components/common/PWAInstallPrompt.tsx

```diff
@@ -50,45 +53,50 @@ export const PWAInstallPrompt: React.FC = () => {
+        <CardHeader className="pb-3">
+          <div className="flex items-start gap-3">
+            <Download className="h-5 w-5 text-blue-400 mt-0.5" />
+            <div className="flex-1">
+              <CardTitle className="text-slate-100 text-sm font-medium">
+                Install Pokemon Save Editor
+              </CardTitle>
+              <CardDescription className="text-slate-300 text-sm mt-1">
+                Get the full app experience with offline support and faster loading.
+              </CardDescription>
+            </div>
+            <CardAction>
+              <Button 
+                variant="ghost" 
+                size="icon"
+                onClick={handleDismiss}
+                className="h-6 w-6 text-slate-400 hover:text-slate-200 hover:bg-slate-700"
+              >
+                <X className="h-4 w-4" />
+              </Button>
+            </CardAction>
           </div>
-          <div className="flex-1 min-w-0">
-            <h3 className="text-sm font-medium text-slate-100">
-              Install Pokemon Save Editor
-            </h3>
-            <p className="text-sm text-slate-300 mt-1">
-              Get the full app experience with offline support and faster loading.
-            </p>
+        </CardHeader>
+        <CardContent className="pt-0">
+          <div className="flex gap-2">
+            <Button
+              onClick={handleInstall}
+              className="flex-1 bg-blue-600 hover:bg-blue-700 text-white text-sm"
+              size="sm"
+            >
+              Install
+            </Button>
+            <Button
+              variant="outline"
+              onClick={handleDismiss}
+              className="flex-1 border-slate-600 text-slate-300 hover:text-slate-100 hover:border-slate-500 hover:bg-slate-700 text-sm"
+              size="sm"
+            >
+              Not now
+            </Button>
           </div>
-          <button
-            onClick={handleDismiss}
-            className="flex-shrink-0 text-slate-400 hover:text-slate-200 transition-colors"
-          >
-            <svg className="w-5 h-5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M6 18L18 6M6 6l12 12" />
-            </svg>
-          </button>
-        </div>
-        <div className="flex gap-2 mt-3">
-          <button
-            onClick={handleInstall}
-            className="flex-1 bg-blue-600 hover:bg-blue-700 text-white px-3 py-1.5 rounded text-sm font-medium transition-colors"
-          >
-            Install
-          </button>
-          <button
-            onClick={handleDismiss}
-            className="flex-1 px-3 py-1.5 border border-slate-600 text-slate-300 hover:text-slate-100 hover:border-slate-500 rounded text-sm transition-colors"
-          >
-            Not now
-          </button>
-        </div>
-      </div>
+        </CardContent>
+      </Card>
     </div>
```

Comment:
do not use shadcn cards, just use shadcn buttons, but keep the style the way you had it before, it looked good the way it was before. with `<div className="bg-slate-800 border border-slate-600 rounded-lg p-4 shadow-lg">` etc

also we have react icons lib installed. use this one `import { HiOutlineCloudDownload } from "react-icons/hi";` looks very similar to the one you used before